### PR TITLE
Hds roadmap 2021

### PR DESCRIPTION
**Pull request by laurakarhu**
## Description

Updates to HDS roadmap, giving a better visibility to coming features and updates

## How Has This Been Tested?
Running docsite locally

### CODE REVIEW

File: site/docs/about/roadmap.mdx [hunk 1]
```diff
@@ -9,79 +9,28 @@ import LargeParagraph from "../../src/components/LargeParagraph";
...
-- ~~Component: Search field~~
-- ~~Pattern: Form validation~~
-- ~~Component: Accordion~~
-- ~~Component: Tabs~~
-
+For previously released features, see changelogs in [What is new](/about/new)
 
 **Q1 2021**
 
+~~- Component: Error template~~
```
Comment (ronijaakkola) [inline site/docs/about/roadmap.mdx:24]: > ~~- Component: Error template~~

Error template has not been implemeted yet.

File: site/docs/about/roadmap.mdx [hunk 2]
```diff
@@ -94,3 +43,23 @@ import LargeParagraph from "../../src/components/LargeParagraph";
 
 
 - **HDS 1.0 Release**
+
+**Q2 2021**
```
Comment (ronijaakkola) [inline site/docs/about/roadmap.mdx:47]: This section renders incorrectly on the actual website (see screenshot, too big spacing between list items). This is due to how docz handles lists. Some extra newlines are often needed to make everything render correctly. Running the doc site locally helps to locate these issues and fix them.

image

File: site/docs/about/roadmap.mdx [hunk 3]
```diff
@@ -94,3 +43,23 @@ import LargeParagraph from "../../src/components/LargeParagraph";
...
+- Sliders
+
+- Pattern: Multi-step forms
+- Pattern: Cookie Consent 
+
+**Q3 2021**
+- Map components
+- Opening hours components
+
+**Q4 2021**
```
Comment (ronijaakkola) [inline site/docs/about/roadmap.mdx:62]: Same with this - it has a different list spacing than other sections.

image

File: site/docs/about/roadmap.mdx [hunk 4]
```diff
@@ -94,3 +43,23 @@ import LargeParagraph from "../../src/components/LargeParagraph";
 
 
 - **HDS 1.0 Release**
+
+**Q2 2021**
+- Image usage instructions
+- Tone of voice instructions
+- Basic tables
```
Comment (ronijaakkola) [inline site/docs/about/roadmap.mdx:50]: These are formatted differently than other sections. 

For example; "Basic tables" vs "Component: Basic tables".

File: site/docs/about/roadmap.mdx [hunk 5]
```diff
@@ -9,79 +9,28 @@ import LargeParagraph from "../../src/components/LargeParagraph";
...
-- ~~Component: Selection group~~
-- ~~Component: Page container~~
-- ~~Component: Search field~~
-- ~~Pattern: Form validation~~
-- ~~Component: Accordion~~
-- ~~Component: Tabs~~
-
+For previously released features, see changelogs in [What is new](/about/new)
 
 **Q1 2021**
```
Comment (ronijaakkola) [inline site/docs/about/roadmap.mdx:22]: Shadow tokens are missing from the list.

File: site/docs/about/roadmap.mdx [hunk 6]
```diff
@@ -9,79 +9,28 @@ import LargeParagraph from "../../src/components/LargeParagraph";
...
-- ~~Component: Tag~~
-- ~~Component: Loading spinner~~
-- ~~Component: Selection group~~
-- ~~Component: Page container~~
-- ~~Component: Search field~~
-- ~~Pattern: Form validation~~
-- ~~Component: Accordion~~
-- ~~Component: Tabs~~
-
+For previously released features, see changelogs in [What is new](/about/new)
```
Comment (ronijaakkola) [inline site/docs/about/roadmap.mdx:20]: For accessibility, HDS doc site often describes links more elaborately.

For example; "For previously released features, see changelogs in the [What is new section](/about/new)."

**Review by ronijaakkola — CHANGES_REQUESTED** [review]

File: site/docs/about/roadmap.mdx (hunk 1 above)
Comment (laurakarhu) [reply site/docs/about/roadmap.mdx:24]: Ah, my bad. I didn't find tickets for it and we have validation patterns including some error handling so I assumed it was included there.

**Review by ronijaakkola — APPROVED** [review]
Looks good and crisp now!

**Pull request merged**